README.md

```diff
@@ -121,7 +121,15 @@ Call `InAppPurchase.purchase` with product id.
 InAppPurchase.purchase(item.productId); // 'rniap.sample.consumable'
 ```
 
-#### 5. Retry
+#### 5. Restore Products
```

Comment:
I think this should be "Restory Purchases" because actually it is not restoring "Product" but "Purchase".
And it would be great if you could write that restore works only for non-consumable and non-expired subscriptions :)

Reply:
We can not restore for specific product. Restore will automatically applies on Products which is Non-renewable, renewable and one time purchase. After success need to call receipt validation to apple server which is done by developer himself